### PR TITLE
Add timeout for install custom kernel

### LINKED ISSUE #515: Add timeout for install custom kernel.
Hit this issue when upgrade kernel in disco, it hung during install package, and our code keep check state in a loop and never exit, we need add a timeout in https://github.com/LIS/LISAv2/blob/master/Libraries/CommonFunctions.psm1#L578

### PR DESCRIPTION
**Pull request by Tong-Xu**
fix https://github.com/LIS/LISAv2/issues/515

### CODE REVIEW

File: Libraries/CommonFunctions.psm1 [hunk 1]
```diff
@@ -573,11 +573,12 @@ function Install-CustomKernel ($CustomKernel, $allVMData, [switch]$RestartAfterU
 				$jobCount += 1
 				#endregion
 			}
-			$packageInstallJobsRunning = $true
 			$kernelInstalledSkipped = $false
 			$kernelMatchSuccess = "CUSTOM_KERNEL_SUCCESS"
 			$customKernelAlreadyInstall="CUSTOM_KERNEL_ALREADY_INSTALLED"
-			while ($packageInstallJobsRunning) {
+			$timeout = New-Timespan -Minutes 180
```
Comment (juhlee-microsoft) [inline Libraries/CommonFunctions.psm1:579]: 3hrs?

**Review by juhlee-microsoft — COMMENTED** [review]
"3hrs"? Or should be 3 minutes?

**Comment by Tong-Xu** [comment]
3 hours . Because when we use  : -CustomKernel linuxnext , We need about 180 minutes.

File: Libraries/CommonFunctions.psm1 (hunk 1 above)
Comment (LiliDeng) [reply Libraries/CommonFunctions.psm1:579]: Why we add the max execution time - we hit infinite loop when hit hung issue during upgrade ubuntu 19.04 on hyperv.
When upgrade into Linux-next in a small size one azure, it needs more than 2 hours, so we put 180 mins.
@iamshital Is it possible to add max execution time when Run-LinuxCmd -RunInBackGround

**Review by juhlee-microsoft — APPROVED** [review]

**Pull request merged**